src/main/java/basic/Exercise1.java

```diff
@@ -5,11 +5,10 @@ public int timUcln(int a, int b){
         if(a == 0 || b == 0)
             return 0;
         if(a % b == 0) {
-            return a > b ? b : a;
+            return Math.min(a, b);
```

Comment:
a % b == 0 thì chỉ có thể là a = 0 hoặc a > b. a = 0 thì không được nên chỉ có thể a>b, vậy chỉ cần
```suggestion
            return a;
```

Reply:
À. Thì ra ex 1 của basic. Dạ rồi để em sửa.Cảm ơn anh

Reply:
A hihi em nghĩ b chứ sao lại là a nhỉ

Reply:
done